scripts/image-builder/build.sh

```diff
@@ -336,23 +341,41 @@ if [ -z "${IMAGE_VERSION}" ] ; then
     exit 1
 fi
 
-# Download openshift local RPM packages (noarch for python and selinux packages)
-rm -rf openshift-local 2>/dev/null || true
-
-OCP_REPO_NAME="rhocp-4.13-for-rhel-${OSVERSION}-$(uname -m)-rpms"
-reposync -n -a "${BUILD_ARCH}" -a noarch --download-path openshift-local \
-    --repo="${OCP_REPO_NAME}" \
-    --repo="fast-datapath-for-rhel-${OSVERSION}-${BUILD_ARCH}-rpms" >/dev/null
+title "Ensure there is at least one osbuild worker running"
+sudo systemctl start osbuild-composer.socket
+sudo systemctl start "osbuild-worker@1.service"
+
+# Enable rhcop-4.13 repo for dependencies
+title "Enabling RHOCP and fast-datapath repositories"
+cat >rhocp-4.13.toml <<EOF
+id = "rhocp-4.13"
+name = "Red Hat OpenShift Container Platform 4.13 for RHEL ${OSVERSION}"
+type = "yum-baseurl"
+url = "https://cdn.redhat.com/content/dist/layered/rhel${OSVERSION}/$(uname -m)/rhocp/4.13/os"
+check_gpg = true
+check_ssl = true
+system = false
+rhsm = true
+EOF
+sudo composer-cli sources add rhocp-4.13.toml
+
+cat >fast-datapath.toml <<EOF
+id = "fast-datapath"
+name = "Fast Datapath for RHEL ${OSVERSION}"
+type = "yum-baseurl"
+url = "https://cdn.redhat.com/content/dist/layered/rhel${OSVERSION}/$(uname -m)/fast-datapath/os"
+check_gpg = true
+check_ssl = true
+system = false
+rhsm = true
+EOF
+sudo composer-cli sources add fast-datapath.toml
 
-# Remove 'microshift' packages to avoid overrides from the remote repository
-find openshift-local -name \*microshift\* -exec rm -f {} \;
-# Exit if no RPM packages were found
-if [ "$(find openshift-local -name '*.rpm' | wc -l)" -eq 0 ] ; then
-    echo "No RPM packages were found at the '${OCP_REPO_NAME}' repository. Exiting..."
-    exit 1
-fi
-createrepo openshift-local >/dev/null
-open_repo_permissions openshift-local
+for name in $(sudo composer-cli sources list); do
+    echo
+    echo "Package source: ${name}"
+    sudo composer-cli sources info "${name}" | sed -e 's/gpgkeys.*/gpgkeys = .../g'
+done
```

Comment:
This may generate a bit noisy output in the current (otherwise clean) sequence. If we'd like to display the enabled repos, can we just display name/url instead of all the parameters? 
```
sudo composer-cli sources info "${name}" | grep -E '^id|^url'
```